packages/env-tests/src/scaffold.ts

```diff
@@ -1,15 +1,39 @@
 import { concurrentMap } from '@celo/base'
+import { CeloContract, ContractKit } from '@celo/contractkit'
+import { newExchange } from '@celo/contractkit/lib/generated/Exchange'
+import { newStableToken } from '@celo/contractkit/lib/generated/StableToken'
+import { ExchangeWrapper } from '@celo/contractkit/lib/wrappers/Exchange'
+import { StableTokenWrapper } from '@celo/contractkit/lib/wrappers/StableTokenWrapper'
 import { generateKeys } from '@celo/utils/lib/account'
 import { privateKeyToAddress } from '@celo/utils/lib/address'
 import BigNumber from 'bignumber.js'
 import { EnvTestContext } from './context'
 
 BigNumber.config({ EXPONENTIAL_AT: 1e9 })
 
-export async function fundAccount(
+export const StableTokenToRegistryName: Record<string, CeloContract> = {
+  cUSD: CeloContract.StableToken,
+  cEUR: 'StableTokenEUR' as CeloContract,
+}
+
+export const ExchangeToRegistryName: Record<string, CeloContract> = {
+  cUSD: CeloContract.Exchange,
+  cEUR: 'ExchangeEUR' as CeloContract,
+}
+
+export async function fundAccountWithcUSD(
   context: EnvTestContext,
   account: TestAccounts,
   value: BigNumber
+) {
+  await fundAccountWithStableTokens(context, account, value, ['cUSD'])
+}
+
+export async function fundAccountWithStableTokens(
+  context: EnvTestContext,
+  account: TestAccounts,
+  value: BigNumber,
+  stableTokensToFund: string[]
```

Comment:
What the rational for funding an account with multiple tokens in the same test? Generally, it's preferable for tests to do as little as possible. For example, if cEUR transfers are broken, all transfer tests will fail if I understand correctly. Consider just specifying one token per test and "invoke the same test" multiple times with a different parameter.

Reply:
Updated funding function to fund account with single stabletoken